App/ProtoControllers/HautResponsable/Utilisateur.php

```diff
@@ -191,10 +191,9 @@ public static function getFormUser($userId = NIL_INT)
     private static function getFormUserSoldes($data, $userId)
     {
         $sql = \includes\SQL::singleton();
-        $config = new \App\Libraries\Configuration($sql);
         $typeAbsencesConges = \App\ProtoControllers\Conge::getTypesAbsences($sql, 'conges');
         if (NIL_INT !== $userId) {
-            $soldesByType = \App\ProtoControllers\Utilisateur::getSoldesEmploye($sql, $config, $userId);
+            $soldesByType = \App\ProtoControllers\Utilisateur::getSoldesEmploye($sql, $userId);
```

Comment:
Codacy found an issue: [Avoid using static access to class '\App\ProtoControllers\Utilisateur' in method 'getFormUserSoldes'.](https://app.codacy.com/manual/Libertempo/web/pullRequest?prid=5122180)